### ISSUE TITLE
prokka

### ISSUE THREAD

**Issue by yolanda-huang** [ubccr/software-layer#182]
https://github.com/tseemann/prokka

**Comment by tonykew** [comment]
Test build completed - more info to follow

@aebruno 
```
tarball:
/srv/cosmos/mirrors/ccr/software-dev/ccr-2023.01-easybuild-tonykew_prokka-1.14.5-gompi-2021b-2024-01-19.185408.tar.gz

git pull for the easyconfig files:
https://github.com/ubccr/software-layer/pull/189

Pavilion test to follow...

```

**Comment by tonykew** [comment]
prokka has been published to the ccrsoft/2023.01 software release

```
vortex1$ module spider prokka
--------------------------------------------------------------------------------------------------------------------------------------------------------------
  prokka: prokka/1.14.5
--------------------------------------------------------------------------------------------------------------------------------------------------------------
    Description:
      Prokka is a software tool for the rapid annotation of prokaryotic genomes.


    You will need to load all module(s) on any one of the lines below before the "prokka/1.14.5" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
vortex1$ 
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 prokka/1.14.5
prokka scripts are located in $EBROOTPROKKA/bin; databases are located in $EBROOTPROKKA/db
vortex1$ 
vortex1$ ls ${EBROOTPROKKA}/bin
prokka                       prokka-biocyc_to_fasta_db  prokka-cdd_to_hmm       prokka-genbank_to_fasta_db  prokka-hamap_to_hmm  prokka-tigrfams_to_hmm
prokka-abricate_to_fasta_db  prokka-build_kingdom_dbs   prokka-clusters_to_hmm  prokka-genpept_to_fasta_db  prokka-make_tarball  prokka-uniprot_to_fasta_db
vortex1$ 
vortex1$ prokka --help
Name:
  Prokka 1.14.5 by Torsten Seemann <torsten.seemann@gmail.com>
Synopsis:
  rapid bacterial genome annotation
Usage:
  prokka [options] <contigs.fasta>
[...]
vortex1$ 
```
